rainforest/rfml.go

```diff
@@ -253,6 +278,13 @@ func (r *RFMLWriter) WriteRFMLTest(test *RFTest) error {
 		}
 	}
 
+	if test.State == "disabled" {
+		_, err = writer.WriteString("# disabled: true\n")
```

Comment:
Why not `state: disabled`? (Defaulting to `enabled`?)

Reply:
For some reason I thought we decided to go with the `# disabled: boolean` route, but I can change it to go by the state.

Reply:
lol there's no tech spec so I don't think we decided anything :smile:. I think going with the API way is better to avoid confusion, and maybe we should leave it out by default when downloading tests to RFML.